crates/rpc/rpc/src/eth/cache/mod.rs

```diff
@@ -51,6 +52,9 @@ type ReceiptsLruCache<L> = MultiConsumerLruCache<H256, Vec<Receipt>, L, Receipts
 
 type EnvLruCache<L> = MultiConsumerLruCache<H256, (CfgEnv, BlockEnv), L, EnvResponseSender>;
 
+/// The max concurrent database operations for `EthStateCacheService`.
+pub const MAX_CONCURRENT_DB_OPS: usize = 512;
```

Comment:
we should move this to

https://github.com/paradigmxyz/reth/blob/8c015c176a6cf541fd81933170cea5b29cb20da9/crates/rpc/rpc/src/eth/cache/config.rs#L27-L27